src/layer/poi.js

```diff
@@ -297,7 +309,11 @@ export const poi = {
         "tram_stop",
       ],
       15,
-      [...getSubclasses(iconDefs.bar), ...getSubclasses(iconDefs.coffee)],
+      [
+        ...getSubclasses(iconDefs.fuel),
```

Comment:
Gas stations are present in the tiles at zoom 14, and I think they're worth highlighting at that zoom level. If the labels look too cluttered, we can give them a higher minimum zoom than the icons, just like we do with bus stops.

Reply:
Done in commit e8c98f3acb8b1571478bb2c6ece6f061f695161c

Reply:
Looks a little better. Though this confirms my worry that the labels are cluttered at low zoom. Here's z15 and z16, respectively:

![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/1732117/f8fbc18e-58ea-4aee-b792-d1b2bd3af5a2)
![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/1732117/b5234117-1eea-45dd-b07a-52a701d76aeb)

The layer's `text-field` is set to an empty string if certain conditions are met, namely if the POI is a `bus_stop` or `tram_stop` and the zoom level is too low. Let's try adding `fuel` to the list on line 338.